src/main/java/com/kodilla/tictactoe/Game.java

```diff
@@ -0,0 +1,126 @@
+            primaryStage.setScene(gameScene);
+        }
+        ));
+
+        Text difficultyLevel = new Text(Tile.getDifficulty());
+        difficultyLevel.setTranslateX(336);
+        difficultyLevel.setTranslateY(187);
+        difficultyLevel.setFont(Font.font("Verdana", 20));
+        difficultyLevel.setFill(Color.WHITE);
+
+        menu.addMenuData(new Pair<>(new Label("POZIOM TRUDNO\u015aCI"), () -> {
+            Tile.changeDifficulty();
+            difficultyLevel.setText(Tile.getDifficulty());
+        }));
+        menu.addMenuData(new Pair<>(new Label("ZAPISZ GR\u0118"), () -> {
+            saveGame.save();
+        }));
+        menu.addMenuData(new Pair<>(new Label("WCZYTAJ GR\u0118"), () -> {
+            saveGame.load();
+            createNewContent();
+            content.restoreBoardAfterLoadGame();
+            Tile.setPlayable(GameState.getGameState().checkState());
+        }));
+
+        Text wonGames = new Text(""+Scores.getWonGames());
+        wonGames.setTranslateX(336);
+        wonGames.setTranslateY(187);
+        wonGames.setFont(Font.font("Verdana", 20));
+        wonGames.setFill(Color.WHITE);
+
+        Text lostGames = new Text(""+Scores.getLostGames());
+        lostGames.setTranslateX(336);
+        lostGames.setTranslateY(247);
+        lostGames.setFont(Font.font("Verdana", 20));
+        lostGames.setFill(Color.WHITE);
+
+        menu.addMenuData(new Pair<>(new Label("STATYSTYKI"), () -> {
+            wonGames.setText(""+Scores.getWonGames());
+            lostGames.setText(""+Scores.getLostGames());
+            primaryStage.setScene(scoreScene);
+        }));
+        menu.addMenuData(new Pair<>(new Label("ZAMKNIJ MENU"), () -> primaryStage.setScene(gameScene)));
+        menu.addMenuData(new Pair<>(new Label("WYJ\u015aCIE Z GRY"), Platform::exit));
+
+        menu.generate();
+        menu.getRoot().getChildren().add(difficultyLevel);
+
+        btnBackToMenu.setOnAction(e -> primaryStage.setScene(menuScene));
+        content.getRoot().getChildren().add(btnBackToMenu);
+
+        // SCORES MENU
+        scores.addMenuData(new Pair<>(new Label("          STATYSTYKI"), () -> {}));
+        scores.addMenuData(new Pair<>(new Label("WYGRANE"), () -> {}));
+        scores.addMenuData(new Pair<>(new Label("PRZEGRANE"), () -> {}));
+        scores.addMenuData(new Pair<>(new Label("RESETUJ STATYSTYKI"), () -> {
+            Scores.reset();
+            wonGames.setText(""+Scores.getWonGames());
+            lostGames.setText(""+Scores.getLostGames());
+        }));
+        scores.addMenuData(new Pair<>(new Label(""), () -> {}));
+        scores.addMenuData(new Pair<>(new Label(""), () -> {}));
+        scores.addMenuData(new Pair<>(new Label("POWR\u00d3T DO MENU"), () -> primaryStage.setScene(menuScene)));
+
+        scores.generate();
+        scores.getRoot().getChildren().addAll(wonGames, lostGames);
+
+        primaryStage.setScene(menuScene);
+        primaryStage.show();
+    }
```

Comment:
spróbuj main menu i scores menu wydzielić do osobnych metod

Reply:
Nie za bardzo wiem jak to zrobić, bo użycie Text wonGames itp przeplata się w kodzie main menu i scores menu

Reply:
to co się przeplata może być parametrem tych metod:
`createScoresMenu(Stage primaryStage, Text wonGames, Text lostGames, Text drawGames)`